renku/ui/service/controllers/cache_migrations_check.py

```diff
@@ -71,11 +71,15 @@ def renku_op(self):
 
     def to_response(self):
         """Execute controller flow and serialize to service response."""
-        return result_response(self.RESPONSE_SERIALIZER, self.execute_op())
+        if "project_id" in self.context:
+            result = self.execute_op()
+        else:
+            # NOTE: use quick flow but fallback to regular flow in case of unexpected exceptions
+            try:
+                result = self._fast_op_without_cache()
+            except (AuthenticationError, ProjectNotFound):
+                raise
+            except Exception:
```

Comment:
```suggestion
            except BaseException:
```

This should be even lower-level and really catch everything. 

Or just this, which amounts to the same thing:
```suggestion
            except:
```

Reply:
Ah, I didn't know about `BaseException` :bulb: 
P.S. I actually went with just `except:` initially, but the linter complained.